packages/scandipwa/src/component/Menu/Menu.component.js

```diff
@@ -197,7 +197,7 @@ export class Menu extends PureComponent {
         const { header_content: { header_cms } = {} } = window.contentConfiguration;
 
         if (header_cms) {
-            return <CmsBlock identifier={ header_cms } />;
+            return <CmsBlock identifier={ header_cms } blockType="header_cms" />;
```

Comment:
Please, store this `"header_cms"` string into a constant so that it can be used later on somewhere else.